bundle/bundle_test.go

```diff
@@ -23,7 +23,7 @@ func TestLoadExists(t *testing.T) {
 
 func TestBundleCacheDir(t *testing.T) {
 	projectDir := t.TempDir()
-	f1, err := os.Create(filepath.Join(projectDir, "bundle.yml"))
```

Comment:
Didn't we agree to databricks.yml? (no "a")

Reply:
I made `databricks.yml` the default but kept support for both.